tardis/model/base.py

```diff
@@ -84,6 +84,32 @@ def r_outer(self):
         return self.geometry.r_outer.values * self.geometry_units["r_outer"]
 
 
+class Composition:
+    """
+    Holds information about model composition
+
+    Parameters
+    ----------
+    density : astropy.units.quantity.Quantity
+    abundance : pd.DataFrame
+    atomic_mass : pandas.core.series.Series
+
+    Attributes
+    ----------
+    number_density : pd.DataFrame
+    """
+
+    def __init__(self, density, abundance, atomic_mass):
+        self.density = density
+        self.abundance = abundance
```

Comment:
Change name of `abundance` to `isotopic_mass_fraction`

Add a new `@property` called `elemental_mass_fraction` which groups all of the isotopes by atomic number and adds up their respective mass fractions

Reply:
Use the class in io/decay to represent isotopic mass fractions